app/test_main.py

```diff
@@ -1,3 +1,88 @@
+import pytest
 from app.main import get_human_age
 
-# write your code here
+
+class TestGetHumanAge:
+    @pytest.mark.parametrize(
+        "initial_cat_age,"
+        "initial_dog_age,"
+        "expected_human_age_cat,"
+        "expected_human_age_dog",
```

Comment:
Unite the `expected_human_age_cat` and `expected_human_age_dog` to one variable `expected_result` where you will store a list of values. It will help to shorten the space and make the code more concise.